plugins/postgresql.json

```diff
@@ -1,19 +1,20 @@
 {
-  "name": "postgresql",
-  "version": "0.0.1",
-  "match": "^postgresql(_[0-9]+)?$",
-  "readme": "To initialize the database run `initdb`.",
-  "env": {
-    "PGDATA": "{{ .Virtenv }}/data",
-    "PGHOST": "{{ .Virtenv }}"
-  },
-  "create_files": {
-    "{{ .Virtenv }}/data": ""
-  },
-  "services": {
-    "postgresql": {
-      "start": "pg_ctl start -l {{ .Virtenv }}/logfile -o \"-k {{ .Virtenv }}\"",
-      "stop": "pg_ctl stop"
+    "name": "postgresql",
+    "version": "0.0.1",
+    "match": "^postgresql(_[0-9]+)?$",
+    "readme": "To initialize the database run `initdb`.",
+    "env": {
+        "PGDATA": "{{ .Virtenv }}/data",
+        "PGHOST": "{{ .Virtenv }}"
+    },
+    "create_files": {
+        "{{ .Virtenv }}/data": "",
+        "{{ .Virtenv }}/process-compose.yaml": "postgresql/process-compose.yaml"
+    },
+    "services": {
+        "postgresql": {
+            "start": "pg_ctl start -l {{ .Virtenv }}/logfile -o \"-k {{ .Virtenv }}\"",
+            "stop": "pg_ctl stop"
```

Comment:
Another option is to turn this into a `process-compose.yaml` automatically so that instead of having

`"{{ .Virtenv }}/process-compose.yaml": "postgresql/process-compose.yaml"` in each plugin, we can generate basic ones as needed and only specify one if it has some custom stuff.

Another pottential improvement is to turn the start and stop commands into env variables, e.g.

```
$POSTGRESQL_START="pg_ctl start -l {{ .Virtenv }}/logfile -o \"-k {{ .Virtenv }}\""
```

and subsequently put that into the process-compose.yaml

this would allow end users to override parts of the process manager without having re-write the process compose.

Reply:
I'll need to leave them for backwards compatibility until we release the process-compose change

I think having a way to specify basic process-compose config in the json would be great. I think we can follow that up on a subsequent release.

Reply:
> Since we now start and stop services only using process-compose, these would no longer be needed. We may need to change how `plugin.Services` works.

In this PR, `plugin.Services` is using process compose and disregarding these commands. It seems to work with Postgres, though I'll probably change it to use the foreground command